### ISSUE TITLE
Authme won't hook to authmebungee

### ISSUE THREAD

**Issue by TheGameSpider** [AuthMe/AuthMeReloaded#2003]
### What behaviour is observed:
After logging in, I cannot use any commands or write chat messages, after setting chatRequiresAuth and commands: requireAuth to false, autoLogin to true, I can use commands but I need to login on each server. BungeeCord hook is set to true on each server.

### What behaviour is expected:
After logging in on my auth server, it should automatically log me in on every server.

### Steps/models to reproduce:
Install papermc 1.12.2, latest bungeecord, authme 5.6.0 b2373, latest authmebungee-2.2.0-snapshot.

### Plugin list:
plugins: authme, essentials, essentialschat, essentialsgeoip, essentialsprotect, essentialsspawn, fastlogin, multiverse-core, permissionsex, pingplayer, placeholderapi, playerpoints, plugman, protocollib, residence, skinsrestorer, supervanish, vault, viaversion, worldguard

authserver plugins: authme, essentialsspawn, fastlogin, protocollib, viaversion.

bungee plugins: authmebungee, fastlogin

### Environment description
bungeecord, auth server, other servers, authme with mysql database.
Logging in with premium account with after using `/cracked` command.

### AuthMe build number:
b2373

### Error Log:
No errors

### Configuration:
**authmebungee:**
```
authServers: 
- lobby
allServersAreAuthServers: false
commands:
    requireAuth: false
    whitelist: 
  - login, register, etc
chatRequiresAuth: false
serverSwitch:
    requiresAuth: false
    kickMessage: Authentication required.
autoLogin: true
sendOnLogout: false
unloggedUserServer: ''
```

**authme:**
sessions disabled
Hooks:
```
Hooks:
    # Do we need to hook with multiverse for spawn checking?
    multiverse: true
    # Do we need to hook with BungeeCord?
    bungeecord: true
    # Send player to this BungeeCord server after register/login
    sendPlayerTo: 'economy'
    # Do we need to disable Essentials SocialSpy on join?
    disableSocialSpy: false
    # Do we need to force /motd Essentials command on join?
    useEssentialsMotd: false
```

**Comment by sgdc3** [comment]
Is "bungeecord" in your spigot.yml files enabled?
Is "ip-forward" in your bungee's config file enabled?

**Comment by TheGameSpider** [comment]
Is "bungeecord" in your spigot.yml files enabled? - Yes
Is "ip-forward" in your bungee's config file enabled? - Yes

It works only if "all servers are auth servers" is set to true

**Comment by sgdc3** [comment]
@TheGameSpider authServers is case sensitive, are you sure that your lobby is called "lobby" and not "Lobby" in your bungee config.yml file?

**Comment by TheGameSpider** [comment]
I am sure.

But the authServers description says that user need to be authenticated to
connect to these servers, doesn't say that these servers are used for
authentication.

Dňa ut, 4. feb 2020, 11:52 Gabriele C. <notifications@github.com>
napísal(a):

**Comment by RedyAu** [comment]
Same here! Have you found any solutions? @TheGameSpider

**Comment by TheGameSpider** [comment]
As I wrote before, authServers are not the servers, which are used for
authentication, but servers, that requires you to be authenticated, so your
Auth server must NOT be on this list.

Dňa st, 1. júl 2020, 15:20 RedyAu <notifications@github.com> napísal(a):

**Comment by RedyAu** [comment]
In the meantime I solved it. I have both my servers on the same MySQL, and have `allServersAreAuthServers: false`. Trough AutheMeBungee, when a player changes servers, plugin messages get sent and they get authenticated instantly.
I also have sessions enabled, so if they change back and forth, they get authenticated doubly, so they get "You have been logged in, as you have logged out in the specified timeframe", "Succesful login" _aand_ also that they are already logged in.
A bit janky but it works.
Maybe something could be done about the tripled login messages, but that's another thing.

**Comment by RedyAu** [comment]
I think this issue can be closed, as everything works perfectly even on 1.16.1.

**Comment by PolloEnCubitos** [comment]
Hi i got a similar problem with this, When first time registering an account i get teleported to my main hub! but when I log in again this time i dont get teleported to my main hub server i just stay on the auth server (I can execute and do all things after logged in but i dont get teleported!

No mysql
allServersAreAuthServers: true
autoLogin: true
Hooks:
bungeecord: true
